TensorFI/faultTypes.py

```diff
@@ -174,3 +174,24 @@ def bitTensor ( dtype, val):
 	val = val.reshape(valShape)
 	return dtype.type( val )
 
+def bitMultiScalar( dtype, val ):
+	"Flip multiple bits of the scalar value"   
+	fiConf = injectFault.getFIConfig()
+	for x in range(fiConf.bitCount):
+		randomBitFlip(val)
```

Comment:
You need to choose the bit positions that are to be flipped in advance before you call `randomBitFlip`, else you risk repetition